integration_test/third_party_apps_data/applications/couchbase/enable

```diff
@@ -7,6 +7,8 @@ metrics:
   receivers:
     couchbase:
       type: couchbase
+      username: admin
```

Comment:
Added username and password as they are required. (Validation fails without them as expected)